contents/docs/session-replay/how-to-control-which-sessions-you-record.mdx

```diff
@@ -24,10 +34,8 @@ posthog.init('<ph_project_api_key>', {
     disable_session_recording: true,
     // ... other options
 })
-posthog.onFeatureFlags(function () {
-    if (posthog.isFeatureEnabled('your-feature-flag')) {
```

Comment:
I wonder if we need a migration guide in case people have this code snippet in their apps, even fine-tuning the cost control, the `startSessionRecording` is always guarded by the former `your-feature-flag`

Reply:
I think the migration is "only" link the flag in config and delete your code. But yep worth calling that out